packages/plugins/rubygem-redhat_access_lib/rubygem-redhat_access_lib.spec

```diff
@@ -0,0 +1,67 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
```

Comment:
This looks like an old template and it's missing various dependencies. Please have a look at the output of `./add_gem_package.sh redhat_access_lib scl foreman-plugins-nightly-rhel7`